x-pack/plugins/synthetics/server/synthetics_service/project_monitor/normalizers/browser_monitor.ts

```diff
@@ -44,27 +44,7 @@ export const getNormalizeBrowserFields = ({
     [ConfigKey.FORM_MONITOR_TYPE]: FormMonitorType.MULTISTEP,
     [ConfigKey.SOURCE_PROJECT_CONTENT]:
       monitor.content || defaultFields[ConfigKey.SOURCE_PROJECT_CONTENT],
-    [ConfigKey.THROTTLING_CONFIG]:
-      typeof monitor.throttling !== 'boolean'
-        ? `${monitor.throttling?.download}d/${monitor.throttling?.upload}u/${monitor.throttling?.latency}l`
-        : defaultFields[ConfigKey.THROTTLING_CONFIG],
-    [ConfigKey.DOWNLOAD_SPEED]: `${
-      typeof monitor.throttling !== 'boolean'
-        ? monitor.throttling?.download
-        : defaultFields[ConfigKey.DOWNLOAD_SPEED]
-    }`,
-    [ConfigKey.UPLOAD_SPEED]: `${
-      typeof monitor.throttling !== 'boolean'
-        ? monitor.throttling?.upload
-        : defaultFields[ConfigKey.UPLOAD_SPEED]
-    }`,
-    [ConfigKey.IS_THROTTLING_ENABLED]:
-      Boolean(monitor.throttling) ?? defaultFields[ConfigKey.IS_THROTTLING_ENABLED],
-    [ConfigKey.LATENCY]: `${
-      typeof monitor.throttling !== 'boolean'
-        ? monitor.throttling?.latency
-        : defaultFields[ConfigKey.LATENCY]
-    }`,
+    [ConfigKey.THROTTLING_CONFIG]: defaultFields[ConfigKey.THROTTLING_CONFIG],
```

Comment:
This needs to be updated, otherwise project monitors will always use default throttling, even when custom throttling is provided. 

On the synthetics agent side, may need to update the schema to provide label when using custom throttling, or even update the synthetics cli to allow the user to specify the various static options.

Please add tests for supplying custom throttling to project monitor normalizers.